src/modules/tas_studio/editor/mod.rs

```diff
@@ -3974,14 +4019,14 @@ fn smoothed_yaws(
 
         // Walk forward half an interval.
         middle_frame_half
-            .chain(zip(
+            .chain(izip!(
```

Comment:
```suggestion
            .chain(zip(
```
This can be rolled back. And import removed.